src/pcre2_compile.c

```diff
@@ -10439,13 +10441,7 @@ if (has_lookbehind)
     }
   memset(cb.groupinfo, 0, (2 * cb.bracount + 1) * sizeof(uint32_t));
   errorcode = check_lookbehinds(cb.parsed_pattern, NULL, NULL, &cb, &loopcount);
-  if (errorcode != 0)
-    {
-    /* BUG: check_lookbehinds() is missing code for a valid META */
-    PCRE2_ASSERT(errorcode != ERR70);
-
-    goto HAD_CB_ERROR;
-    }
```

Comment:
I would go to the assertion direction. Has many advantages.

Reply:
this assert is an open coded conditional "UNREACHABLE"  which would only trigger if the errorcode that was triggered is an internal error.

Reply:
I know what it does. But this affect release code. Asserts are better. For me the "it is possible to be wrong" cases are equal, and it is not worth put a lot of release asserts because of them.

Reply:
I moved the assertion, rather than deleting it. We now assert at the point where we generate the ERR70, rather than requiring the caller to check for ERR70. Much better - callers shouldn't have to remember to check for internal errors in low-level functions.

Reply:
> We now assert at the point where we generate the ERR70

Did that commit went missing?